packages/admin/admin-rte/.eslintrc.json

```diff
@@ -4,6 +4,18 @@
     "rules": {
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
-        "@comet/no-other-module-relative-import": "off"
+        "@comet/no-other-module-relative-import": "off",
+        "react/react-in-jsx-scope": "off",
+        "no-restricted-imports": [
+            "error",
+            {
+                "paths": [
+                    {
+                        "name": "react",
+                        "importNames": ["default"]
+                    }
+                ]
+            }
+        ]
```

Comment:
We could add this to all packages, am I right? Just not the Demo because of the admin generator

So you could also add it to https://github.com/vivid-planet/comet/pull/2436/files and https://github.com/vivid-planet/comet/pull/2440/files (and everywhere else where it's missing)

Reply:
True, we will add the rule in a separate PR where possible.